Homework-7/Validations/CountryCheckAttribute.cs

```diff
@@ -0,0 +1,23 @@
+﻿
+using System.ComponentModel.DataAnnotations;
+using static Homework_7.Enum.Country;
+
+namespace Homework_7.Validations
+{
+    public class CountryCheckAttribute : ValidationAttribute
+    {
+        protected override ValidationResult IsValid(object value, ValidationContext validationContext)
+        {
+            if (value != "" &&value != null)
+            {
+                if (System.Enum.TryParse(typeof(CountryType), value.ToString(), out object result))
+                {
+                    return ValidationResult.Success;
+                }
+                return new ValidationResult("Invalid country value.");
```

Comment:
Response messages to user should be clear. Eg: `Invalid country value. Country must be one of the following values: \"USA\", \"Canada\", \"UK\", \"Australia\"`